testapps/testapp/src/main/AndroidManifest.xml

```diff
@@ -51,7 +51,8 @@
             android:name="com.microsoft.identity.client.testapp.StartActivity"
             android:taskAffinity=""
             android:theme="@style/AppTheme.NoActionBar"
-            android:windowSoftInputMode="stateHidden">
+            android:windowSoftInputMode="stateHidden"
+            android:exported="true">
```

Comment:
does the calling app need to make any changes for these exported activities?

Reply:
There is no action item on calling apps when they update MSAL with my changes. But, when they update the compileSDK version to 34 in their app, they need to ensure that the BrowserTabActivity is exported and is true. Otherwise, we cannot get back the token.